### DISCUSSION TITLE
Panasonic Lumix DC-TZ200D to the database?

### DISCUSSION THREAD

**Discussion by JS-Fi** [lensfun/lensfun#2386] [General]
Hello,

I noticed that the camera is not in the database. However, technically same camera, DC-TZ200 is in the database. So I’m asking, could the DC-TZ200D be added to the database as well? They share identical lens and optics. Panasonic only updated the camera’s screen on the D model. Afaik, also different region variants shares the same thing, for example DC-ZS200 / DC-ZS200D. So the D models can use the same profile, only if it got detected correctly.

For example, at the moment for this reason, Affinity Photo does not detect a correct lens profile for the camera, and especially on the Affinity Photo’s iPad version, which is more limited, this causes more trouble, as you cant choose the lens profile on the raw import view. And I learned that Affinity uses LensFun as their database.

Thanks if this can be done!

Best regards,
J

**Comment by sarunasb** [comment]
@JS-Fi Can you please provide a RAW sample from the ”D” model. Thank you.

**Reply by JS-Fi** [reply]
[Raw_Sample.zip](https://github.com/user-attachments/files/18220337/Raw_Sample.zip)
Here is one.
